homeassistant/components/sensor/miflora.py

```diff
@@ -31,9 +29,7 @@
 DEFAULT_NAME = 'Mi Flora'
 DEFAULT_RETRIES = 2
 DEFAULT_TIMEOUT = 10
-
-UPDATE_INTERVAL = 1200
-MIN_TIME_BETWEEN_UPDATES = timedelta(seconds=UPDATE_INTERVAL)
```

Comment:
Hmm, was not aware of `SCAN_INTERVAL`. That would also make it possible to set `force_update` to true, the case right now will be a new state event being fired every 30 seconds with resulting writes to the database.

Have I understood it correctly that the `SCAN_INTERVAL` defines the behaviour for the whole platform? Not for each device? And that `CONF_CACHE` is device-specific (in the miflora platform)?